main.go

```diff
@@ -22,7 +22,13 @@ type Params struct {
 	Username      *string  `json:"username,omitempty"`
 }
 
+var (
+	buildDate string
```

Comment:
For now I would keep it and over the time we can see if it makes sense to do some kind of versioning or if we simply add the commit sha.